.ci_support/linux_ppc64le_.yaml

```diff
@@ -0,0 +1,19 @@
+c_compiler:
+- gcc
+c_compiler_version:
+- '8'
```

Comment:
Curious that we get `8` here and [`7` on `x86_64`]( https://github.com/conda-forge/cudatoolkit-feedstock/blob/815aa53361669ba010412a441d710aca027e20df/.ci_support/linux_64_.yaml#L3-L4 ). Don't think this should be a big issue here, but it might matter more for feedstocks compiling CUDA kernels.

Reply:
I think that must be coming from pinnings, right? IIRC Power8 needs a higher version of GCC

Reply:
IIRC this will limit nvcc support to CUDA 10.1 or greater. Ideally we can someone handle these constraints in the cudatoolkit recipe to prevent getting an incompatible CUDA version?

Reply:
Right now we are only packaging CUDA 11.0. Is this ok for that version?

Though at some point we are going to want to go back and add the older versions (back to 9.2). IIUC we will want to constrain those right? How should we be constraining them?

Reply:
Yes, for nvcc in CUDA 11.0 it supports up to gcc 9.x so this should be fine. For nvcc in CUDA 11.1 it supports up to gcc 10.x.

Yes we will want to constrain those versions to prevent getting newer host compilers than what is supported. There's a support table here: https://gist.github.com/ax3l/9489132#nvcc

Note that we may want to break out the `nvcc_linux-64` feedstock scripts into a separate package so that if someone wants to build against cuda libraries without actually using `nvcc` (typical for libraries without kernels), we should enable a path for them to continue to use newer host compilers.

Reply:
It probably makes more sense to include these compiler constraints in the `nvcc` package as opposed the `cudatoolkit` package as this is a constraint for when someone goes to compile CUDA kernels as opposed to when a user installs prebuilt binaries. In the latter case, the hope is we already handled this correctly in conda-forge when building packages and we really shouldn't need compilers in the end user's install.